web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -146,7 +146,8 @@ goog.require('gn_alert');
           // Full text on all fields
           // 'queryBase': '${any}',
           // Full text but more boost on title match
-          'queryBase': 'any:(${any}) resourceTitleObject.\\*:(${any})^2',
+          'queryBase': 'anyger:(${any})',
```

Comment:
So here we have to define a strategy on which fields to search on now.

I see a couple of options depending on the type of catalogue content (and based on what we did in the past):
a) catalogue admin set queryBase on a specific field which should not change - eg. when the admin knows that the catalogue is all in one language 
b) UI lang configuration - if user interface is in french, search on `anyfre` and `resourceTitleObject.langfre` - Most of the time the list of UI language is limited to the list of languages used in records
c) add a language detection of the input (something like https://www.npmjs.com/package/languagedetect) and choose which field to search on. If no results, we can suggest to search on all languages

Reply:
Let's do it in 2 steps.
* First this PR will add per field analyzer for a set of languages that we can test on various situation
* Then we address the search strategies in other PRs.

I'll change the querybase to use `any.*` for now.